gr-uhd/lib/usrp_source_impl.cc

```diff
@@ -203,6 +203,65 @@ ::uhd::gain_range_t usrp_source_impl::get_gain_range(const std::string& name, si
     return _dev->get_rx_gain_range(name, chan);
 }
 
+bool usrp_source_impl::has_power_reference(size_t chan)
+{
+#ifdef UHD_USRP_MULTI_USRP_POWER_LEVEL
+    if (chan >= _stream_args.channels.size()) {
```

Comment:
Please also use these "chan" checks in "usrp_sink_impl.cc"

Reply:
Oh yikes. Thanks for the patient review. That was clearly done in haste.